airbyte-integrations/connectors/source-alloydb/src/main/java/io/airbyte/integrations/source/alloydb/AlloyDbSource.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.source.alloydb;
+
+import io.airbyte.integrations.base.IntegrationRunner;
+import io.airbyte.integrations.base.Source;
+import io.airbyte.integrations.source.postgres.PostgresSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This file will soon be removed. Any change to this file should also be duplicated to
```

Comment:
this comment is a bit confusing, probably we can change it or remove at all?

Reply:
done